content/pt-br/docs/reference/access-authn-authz/abac.md

```diff
@@ -0,0 +1,137 @@
+* `/version` para recuperar a versão do servidor via `kubectl version`.
+* `/swaggerapi/*` para operações de criação/atualização.
+
+Para inspecionar as chamadas HTTP envolvidas em uma operação kubectl específica, você pode ativar aumentar a verbosidade:
+
+     kubectl --v=8 versão
+
+## Exemplos
+
+ 1. Alice pode fazer qualquer coisa em todos os recursos:
+
+    ```json
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "alice", "namespace": "*", "resource": "*", "apiGroup": "*"}}
+    ```
+ 2. O Kubelet pode ler qualquer pod:
+
+    ```json
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "kubelet", "namespace": "*", "resource": "pods", "readonly": true}}
+    ```
+ 3. O Kubelet pode ler e escrever eventos:
+
+    ```json
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "kubelet", "namespace": "*", "resource": "events"}}
+    ```
+ 4. Bob pode ler pods somente pertencentes ao namespace "projectCaribou":
+
+    ```json
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"user": "bob", "namespace": "projectCaribou", "resource": "pods", "readonly": true}}
+    ```
+ 5. Qualquer pessoa pode realizar requisições de somente-leitura em todos os caminhos que não são de recursos:  
+
+    ```json
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"group": "system:authenticated", "readonly": true, "nonResourcePath": "*"}}
+    {"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"group": "system:unauthenticated", "readonly": true, "nonResourcePath": "*"}}
+    ```
+
+[Exemplo de arquivo completo](https://releases.k8s.io/v{{< skew currentPatchVersion >}}/pkg/auth/authorizer/abac/example_policy_file.jsonl)
+
+## Uma rápida observação sobre contas de serviço
+
+Cada conta de serviço tem um nome de usuário ABAC correspondente, e o nome de usuário dessa conta de serviço é gerado de acordo com a convenção de nomenclatura:
+
+```shell
+system:serviceaccount:<namespace>:<serviceaccountname>
+```
+
+A criação de um novo namespace leva à criação de uma nova conta de serviço no seguinte formato:
+
+```shell
+system:serviceaccount:<namespace>:default
+```
+
+For example, if you wanted to grant the default service account (in the `kube-system` namespace) full 
+privilege to the API using ABAC, you would add this line to your policy file:
+
+Por exemplo, se você quiser conceder à conta de serviço padrão (no namespace `kube-system`) privilégio total à API usando ABAC, adicione esta linha ao seu arquivo de política:
+
+```json
+{"apiVersion":"abac.authorization.kubernetes.io/v1beta1","kind":"Policy","spec":{"user":"system:serviceaccount:kube-system:default","namespace":"*","resource":"*","apiGroup":"*"}}
+```
+
+O servidor de API precisará ser reiniciado para selecionar as novas linhas da política.
```

Comment:
```suggestion
O servidor de API precisará ser reiniciado para carregar as novas linhas da política.
```

Suggestion for readability.

Reply:
Tried to addressed in [929685e](https://github.com/kubernetes/website/pull/41800/commits/929685eb8a05d2319710c87419e0da1829aef706), let me know if I still left something out.